t/50file-config.t

```diff
@@ -2,6 +2,7 @@ use strict;
 use warnings;
 use Test::More;
 use t::Util;
+use Gzip::Faster qw/gunzip_file/;
```

Comment:
Please use `gzip -cd < file | wc -c` or something similar instead of adding a new dependency.